light-cli/src/commands/balance/index.ts

```diff
@@ -0,0 +1,136 @@
+import { Command, Flags } from "@oclif/core";
+import { User, Balance, InboxBalance, Utxo } from "@lightprotocol/zk.js";
+import { CustomLoader, getUser } from "../../utils/utils";
+
+class BalanceCommand extends Command {
+  static description =
+    "Retrieve the balance, inbox balance, or UTXOs for the user";
+
+  static flags = {
+    balance: Flags.boolean({
+      char: "b",
+      description: "Retrieve the balance",
+      default: false,
+    }),
+    inbox: Flags.boolean({
+      char: "i",
+      description: "Retrieve the inbox balance",
+      default: false,
+    }),
+    utxos: Flags.boolean({
+      char: "u",
+      description: "Retrieve the UTXOs",
+      default: false,
+    }),
+    inboxUtxos: Flags.boolean({
+      char: "x",
+      description: "Retrieve the inbox UTXOs",
+      default: false,
+    }),
+    latest: Flags.boolean({
+      char: "l",
+      description: "Retrieve the latest balance, inbox balance, or UTXOs",
+      default: true,
+    }),
+  };
+
+  protected finally(_: Error | undefined): Promise<any> {
+    process.exit();
+  }
+
+  static examples = [
+    "$ light balance --balance",
```

Comment:
Let's return the normal and inbox balances by default, then we filter with an extra flag for the specific balances.
I think we need a flag to filter for assets
```
light balance --token usdc
```
should display only usdc utxos as follows:
(inbox can be skipped if zero)
(since this is a repetitive pattern you can just write a function which you can reuse for all balances)
USDC:
Balance:
total amount: 100,00
total sol amount: 0,0001
number of utxos: 2
Utxos:
 1. usdc: 90 sol: 0.0005 commitmentHash: <hash123>
 2. usdc: 10 sol: 0.0005 commitmentHash: <hash124>
Inbox Balance:
total amount: 50,00
total sol amount: 0
number of utxos: 1
Utxos:
 1. usdc: 50 sol: 0  commitmentHash: <hash123>

Reply:
in a way you already did that with the log utxos function

Reply:
it probably makes sense to add a stringify option which returns the utxo as a string

Reply:
and a verbose flag which displays every field of a utxo